wrapper:
  config_file_name: avia.yaml
  lidar_topic: /livox/pointcloud
  imu_topic: /livox/imu
  lidar_type: 0 # 0)avia 1) velodyne

front_end:
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  extrin_t: [0.4165,0.2326,-0.2840]
  filter_leaf_size: 0.5f

ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001

map:
  map_side_length_2: #地图边长的一半
  map_resolution: #地图分辨率